src/apps/chifra/pkg/types/types_function_test.go

```diff
@@ -0,0 +1,74 @@
+package types
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+)
+
+func TestFunctionToAbiMethod(t *testing.T) {
+	input := &SimpleFunction{
```

Comment:
Thanks for taking the time to write tests. I appreciate it a lot. Makes me feel comfortable.

Reply:
It makes my anxiety go away 😄